docs/requirements.txt

```diff
@@ -0,0 +1,12 @@
+# docs/requirements.txt -- required by Read the Docs
+-e .                     # install local package (for autodoc)
+
+# Core Sphinx
+sphinx>=7.2,<8
+sphinx-rtd-theme>=2.0,<3
+
+# Extensions
+myst-parser>=2.0,<3
```

Comment:
I suspect it is the additional imports that are not being supported by readthedocs.io.